cmd/solizard/solizard.go

```diff
@@ -133,14 +136,20 @@ func Run() error {
 				Data:     input,
 			})
 			signedTx, err := types.SignTx(unsignedTx, types.NewEIP155Signer(sctx.ChainId()), sctx.PrivateKey())
+			if err != nil {
+				fmt.Printf("failed to SignTx (reason: %v), maybe rpc is not working.\n", err)
```

Comment:
```suggestion
				fmt.Printf("failed to SignTx (reason: %v)\n", err)
```

@minias Signing is not related with RPC (no network connection required)